.vault-config/dotneteng-status-prod.yaml

```diff
@@ -39,3 +39,8 @@ secrets:
         name: nethelix-prod-kusto-ad-application
       dataSource: https://ingest-engsrvprod.kusto.windows.net:443
       additionalParameters: Streaming=true;Authority Id=72f988bf-86f1-41af-91ab-2d7cd011db47
+
+  build-monitor-hook-dotnet-eng-status-token:
```

Comment:
Is this part of adding the pipelines to build monitor or something extra?

Reply:
This adds a secret that's always been around to secret manager for tracking.

Build Monitor is driven by "build complete" notifications delivered from a service hook. This is the API token to authenticate to our service. It's always been around but was never added to secret manager. Thus it also expired without anyone noticing.